tdrs-backend/tdpservice/data_files/migrations/0011_alter_datafile_section.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-06-08 20:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_files', '0010_legacyfiletransfer'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='datafile',
+            name='section',
+            field=models.CharField(choices=[('Tribal Closed Case Data', 'Tribal Closed Case Data'), ('Tribal Active Case Data', 'Tribal Active Case Data'), ('Tribal Aggregate Data', 'Tribal Aggregate Data'), ('Tribal Stratum Data', 'Tribal Stratum Data'), ('SSP Aggregate Data', 'Ssp Aggregate Data'), ('SSP Closed Case Data', 'Ssp Closed Case Data'), ('SSP Active Case Data', 'Ssp Active Case Data'), ('SSP Stratum Data', 'Ssp Stratum Data'), ('Active Case Data', 'Active Case Data'), ('Closed Case Data', 'Closed Case Data'), ('Aggregate Data', 'Aggregate Data'), ('Stratum Data', 'Stratum Data')], max_length=32),
```

Comment:
Curious why linter doesn't alert on length here. Should this be split across multiple lines?

Reply:
Linter does not look at migrations

Reply:
Frequently these are generated, and don't follow the normal rules.